scripts/bench-hyperfine.sh

```diff
@@ -90,7 +90,7 @@ run_bench() {
         -o "$OUTPUT_DIR/$base_name-march-native" \
         >> /dev/stderr
 
-    CAIRO_NATIVE_RUNTIME_LIBDIR="$ROOT_DIR/target/release" hyperfine \
+    CAIRO_NATIVE_RUNTIME_LIBRARY="$ROOT_DIR/target/release/libcairo_native_runtime.a" hyperfine \
```

Comment:
It would be better to remove this env var from here in its entirely, because the CI has this library on another place, and we already set it, so its better to leave it to the user instead of forcing it here.

Before it didn't affect the CI because this env var is actually unused.